### DISCUSSION TITLE
Do DXVK winetricks do anything for 32bit D3D9 games? Also what's the setup steps?

### DISCUSSION THREAD

**Discussion by i0ntempest** [The-Wineskin-Project/WineskinServer#273] [Q&A] [answered]
Do I just run `dxvk1101_macOS` winetrick? What about `moltenvk`?
I'm having trouble with an old game with vsync, with vsync enabled I get hugely inconsistent frame rates (not really stutter, just framerate going on a rollercoaster).

**Comment by Gcenx** [comment, marked as answer]
> Do DXVK winetricks do anything for 32bit D3D9 games?

Nope, currently only wine64 supports Vulkan via MoltenVK.

Some older games simply won’t run decently via wine32on64 this may or may not improve in CX22 (nightly seem to indicate no improvements)

For example Red Alert2 requires the use of cnc-ddraw to become playable but that’s also the case with upstream wine.

The only thing todo is looking into if there’s another solution that doesn’t require using DXVK or some other dx9>dx10/11 type wrappers.

**Reply by i0ntempest** [reply]
I found this which I tried and works on Windows: https://github.com/elishacloud/dxwrapper
But I can't seem to get it to work in a wrapper. Any extra steps I need to follow to get d3d9 dlls to load?

**Reply by Gcenx** [reply]
If you’ve added say `d3d9.dll` and the other support dlls into the games directory you’ll also need to set `d3d9.dll` to `native,builtin` this can be done via winecfg.

**Reply by i0ntempest** [reply]
K thanks, it worked but it kinda created more problems... Guess there's really no way for this game.
